File: mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int x;
    do
    {
        x = get_int("dimension ");
    }
    while (x <= 0 || x > 8);


        for(int l=0; l<x; l++)
    {
        for (int m=x-1;m>l; m--)
        {
          printf(" ");
        }
        for(int m=0; m<=l;m++)
        {
            printf("#");
        }
        printf("  ");
        for(int m=0; m<=l;m++)
        {
            printf("#");
        }
     printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int
while True:
  height = get_int('What are the dimensions of the pyramid? \n')
  if(height>0 and height<=8):
    break
for x in range(height):
  for l in range(height-1, x, -1):
    print(" ", end="")
  for m in range(0, x+1, 1):
    print("#" , end="")
  print('  ', end='')
  for n in range(x+1):
    print('#', end='')
  print('')